design-document.md

```diff
@@ -106,6 +123,132 @@ to achieve these goals:
    function output itself. {{questions-and-params}} discusses the leakage profiles of 
    various aggregation functions in more detail.
 
+### Threat model
```

Comment:
I think you're right that there should be a section for an attacker on the network. However I find it hard to reason about a threat like the one in your example because we haven't yet sketched out how aggregator discovery works. But certainly we can already call out the necessity of confidential and mutually authenticated links between participating servers.